systest/21million/ludicrous/docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
       source: $GOPATH/bin
```

Comment:
Can this bind point be removed?

Reply:
Good catch, thanks - this should be removed